core/src/main/java/edu/jhuapl/dorset/Application.java

```diff
@@ -71,7 +73,7 @@
      * <p>
      * Uses a null reporter that ignores new reports.
      *
-     * @param router  a router that finds the appropriate agent for a request
+     * @param router a router that finds the appropriate agent for a request
```

Comment:
The extra space between variable name and description is there on purpose. I'm trying to use this standard of JavaDoc: http://blog.joda.org/2012/11/javadoc-coding-standards.html